civictechprojects/views.py

```diff
@@ -116,10 +116,15 @@ def home(request):
 
 def index(request):
     template = loader.get_template('new_index.html')
-    context = {}
+    context = {'userID': request.user.id}
     return HttpResponse(template.render(context, request))
 
 
+def my_projects(request):
+    projects = Project.objects.filter(project_volunteers__id=request.user.id)
```

Comment:
Let's just have this pull from owners right now, since Volunteering won't be implemented this release.